src/components/AnnotationKeyboard.jsx

```diff
@@ -48,6 +48,9 @@ class AnnotationKeyboard extends React.Component {
     const showScript = !this.props.showModern ? `char-button ${letter.name}` : '';
     const characterRep = letter.unicode ? letter.unicode : letter.character;
     const styles = {};
+    const letterInScript = this.props.activeScript.letters.indexOf(letter.name) >= 0;
```

Comment:
While this technically works, it's good practice to check for existence - `this.props.activeScript && this.props.activeScript.letters && this.props.activeScript.letters.indexOf(letter.name) >= 0;`